src/rabbitmq/README.md

```diff
@@ -0,0 +1,48 @@
+# Getting Started:
+### Installation
+- Follow instructions at https://www.rabbitmq.com/install-debian.html#apt to install Erlang and RabbitMQ make sure to install for the correct linux distribution!
+
+### RabbitMQ Server Commands
+- Start server: `sudo systemctl start rabbitmq-server`
+- Check server status: `sudo systemctl status rabbitmq-server`
+- Stop server: `sudo systemctl stop rabbitmq-server`
+
+### Create an admin RabbitMQ User:
+1. `sudo rabbitmqctl add_user admin password`
+2. `sudo rabbitmqctl set_user_tags admin administrator`
+3. `sudo rabbitmqctl set_permissions -p / admin ".*" ".*" ".*"`
+
```

Comment:
I will need to figure out how to configure RabbitMQ on heroku soon, and I can add that to this readme as well.

Reply:
Yep. Though the permissions should be further restricted for safety/security purposes.